cypress/e2e/layers-selection/layers-selection-catalog.cy.ts

```diff
@@ -67,7 +67,11 @@ describe('Catalogue', () => {
         const layers = (<AUTWindowOlMap>window).olMap
           .getLayers()
           .getArray()
-          .filter((l: any) => l.get('cyLayerType') !== 'featureLayer')
+          .filter(
+            (l: any) =>
+              // regex match
```

Comment:
yes we know this is a regex, what is the purpose of this comment?

Reply:
it helps reading, since nothing tells, this is a regex, I did not know this obscure syntax represents a regex